app/controllers/backgrounds_controller.rb

```diff
@@ -41,7 +41,7 @@ def create
   # PATCH/PUT /backgrounds/1
   # PATCH/PUT /backgrounds/1.json
   def update
-    current_data = @background.as_json(include: [{ pipeline_outputs: { only: [:id, :pipeline_run_id] } },
+    current_data = @background.to_json(include: [{ pipeline_outputs: { only: [:id, :pipeline_run_id] } },
```

Comment:
ok, changed that

how do i test it?   does it run on every background modification?

Reply:
Yes - you can test it by modifying any background (go to localhost:3000/backgrounds/:id/edit) and checking that an ArchivedBackground record gets created correctly

Reply:
thanks, confirmed it works